testkit-backend/backend.go

```diff
@@ -484,6 +484,64 @@ func (b *backend) handleRequest(req map[string]any) {
 			"protocolVersion": fmt.Sprintf("%d.%d", protocolVersion.Major, protocolVersion.Minor),
 		})
 
+	case "ExecuteQuery":
+		driver := b.drivers[data["driverId"].(string)]
+		var configurers []neo4j.ExecuteQueryConfigurationOption
+		if rawConfig := data["config"]; rawConfig != nil {
+			executeQueryConfig := rawConfig.(map[string]any)
+			configurers = append(configurers, func(config *neo4j.ExecuteQueryConfiguration) {
```

Comment:
Why is `configurers` an array but you only append once? How about having always one configurer that's just a noop (e.g., returns straight away) if `rawConfig := data["config"]` is `nil`? Might be easier to read, I'm not entirely sure.

Reply:
It's easier to turn an empty array into a vararg than `nil` (there is no ternary syntax for if-else in Go so I picked the slightly-stranger-but-less-verbose option here)